public/lib/column-defaults.js

```diff
@@ -311,6 +311,17 @@ const columnDefaults = [{
     isNew: true,
     isSortable: true,
     sortField: ['statusInPrint']
+},{
+    name: 'commissionedLength',
+    prettyName: 'Commissioned Length',
+    labelHTML: 'Commissioned Length',
+    colspan: 1,
+    title: '',
+    templateUrl: templateRoot + 'commissionedLength.html',
+    template: commissionedLengthTemplate,
+    active: true,
+    isSortable: true,
+    sortField: ['missingCommissionedLengthReason', 'commissionedLength']
```

Comment:
If you order the Commissioned Length field by ascending, you should see missingCommissionedLengthReason first.

Would be nice if we could order by descending to see very high Commissioned Length values, but this just shows empty values as highest. Perhaps some refactoring of the sort would help? (Should blank values be included or not in sort is the bigger question).